app/controllers/application_controller.rb

```diff
@@ -1,11 +1,11 @@
 # Filters added to this controller apply to all controllers in the application.
 # Likewise, all the methods added will be available for all controllers.
-
+require 'will_paginate/array'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.